source/options/feature-list.tsx

```diff
@@ -12,7 +12,7 @@ function moveDisabledFeaturesToTop(): void {
 	const container = $('.js-features');
 	const features = $$('.feature').toSorted((a, b) => a.dataset.text!.localeCompare(b.dataset.text!));
 	const grouped = Object.groupBy(features, feature => elementExists(':checked', feature) ? 'enabled' : 'disabled');
-	for (const group of [grouped.disabled, grouped.enabled]) {
+	for (const group of [grouped.disabled, grouped.enabled].filter(Boolean)) {
```

Comment:
Off, https://github.com/refined-github/refined-github/pull/7831 broke first installs. Good thing I tested in a new browser